ubsan_arangodb_suppressions.txt

```diff
@@ -22,7 +22,10 @@ vptr:s2cell.h
 vptr:s2cell.cc
 
 # fix some V8 false positive
+# TODO(MBkkt) I don't think it's false positive, but probably issue in our version of V8, my issue about arangodbtests
 vptr:v8::Platform
+# TODO(MBkkt) Some issue in our version of V8, only in arangodbtests
+vptr:v8::internal::Isolate::~Isolate
```

Comment:
https://github.com/arangodb/oskar/pull/470
We don't have vptr checks now, but I want to enable them, so add this suppressions